.rubocop_todo.yml

```diff
@@ -1,12 +1,20 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2017-08-01 11:56:46 -0700 using RuboCop version 0.49.1.
+# on 2017-08-16 16:23:48 -0700 using RuboCop version 0.49.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
 # Offense count: 1
+Metrics/AbcSize:
+  Max: 24
```

Comment:
I'd prefer we correct this rather than upping the limit.  I believe this can be accomplished with a few of the suggestions below.